src/lib/ast.rs

```diff
@@ -651,7 +651,6 @@ impl<U: PartialEq + Copy> NodeId<U> {
                                                  current_node.char_no,
                                                  current_node.token_len);
         // Collect all the children
```

Comment:
Does this comment still apply? It looks like the LOC it referred to has been removed?